packages/web-components/src/components/video-player/__stories__/video-player.stories.ts

```diff
@@ -29,6 +30,20 @@ export const aspectRatio4x3 = ({ parameters }) => {
   `;
 };
 
+export const withLightboxMediaViewer = ({ parameters }) => {
+  const { videoId, aspectRatio } = parameters?.props?.VideoPlayer ?? {};
+  return html`
+    <dds-lightbox-video-player-container
+      render-thumbnail
+      player-mode="thumbnail"
```

Comment:
Did we create a whole new component for this? We'll probably need functional specs, and design specs separated out for this if this is the direction and probably even its own category in storybook. Couldn't we just add an option to the existing video player component?

```javascript
<dds-lightbox-video-player-container
      render-thumbnail
      player-mode="lightbox" <------ could we change this here?
      hide-caption
      video-id=${videoId}
      aspect-ratio=${aspectRatio}
    >
</dds-lightbox-video-player-container>
```

Side note, but what's the difference between `player-mode` and `render-thumbnail`?